docs/index.html

```diff
@@ -51,6 +51,13 @@
 
   <div id="app">Loading...</div>
 
+    <div id="banner" style="display: none;">
+      <div id="banner-content">
+        <a href="https://archive-docs.ml5js.org/#/" target="_blank">💜 Looking for the old ml5.js reference website? Find it
+          here!</a>
```

Comment:
Some tiny language tweaks and maybe take the heart out of the link? 

```suggestion
        💜 <a href="https://archive-docs.ml5js.org/#/" target="_blank">Looking for the old ml5.js reference (v0.12.2) Find it 
        here!</a>
```